src/features/game/types/chests.ts

```diff
@@ -12,68 +12,7 @@ export type ChestReward = {
 const multiplier = 900;
 
 const SEASONAL_REWARDS: (weight: number) => ChestReward[] = (weight) => {
-  const isPharaohSeason =
-    Date.now() >= SEASONS["Pharaoh's Treasure"].startDate.getTime() &&
-    Date.now() < SEASONS["Pharaoh's Treasure"].endDate.getTime();
-
-  if (isPharaohSeason) {
```

Comment:
Note:
I'm waiting on the `isBullRun` changes to rebase against so that the `isPharaohSeason` structure can be used as a basis if desired.

Reply:
Once #4762 goes in I can update and move forwad.